src/components/AddressDetailExplorer.js

```diff
@@ -434,6 +435,56 @@ class AddressDetailExplorer extends React.Component {
     return Math.ceil(this.state.balance.transactions / TX_COUNT);
   }
 
+  onNextPageClicked = async () => {
+    this.setState({ loadingPagination: true });
+
+    const transactions = this.state.transactions;
+    const { timestamp, tx_id } = transactions.at(transactions.length - 1);
+
+    const nextPage = this.state.page + 1;
+
+    const lastTx = tx_id;
+    const lastTs = timestamp;
+
+    // Calculate the next page searchAfter, so it we can click previous and come back to it.
+    const searchAfter = {
+      lastTx,
```

Comment:
We are not adding the `lastTx` and `lastTs` as parameters in the URL, right?

Didn't the design state that we should update it?

> Also, the URL should change while paginating with the search_after params (the timestamp and tx_id from the last page)

I think it's important to add them in the URL because it makes the URL shareable. What do you think?